.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Let's omit this? Editor-specific configs shouldn't be committed.

Reply:
hey, yeah there is such practice 
also this is debugger setup which can be very handy if anyone would like to run/debug tests on their own

even in other editor there are very similar configs

Reply:
that's not prefs, that's debug config...

Reply:
Is it something more exotic than go test?
If so, I'd suggest including in an editor-agnostic fashion. If not, then there's minimal value in including it

Reply:
Eh, I don't care about it that much 🤷

Reply:
do you have any reasons to not include it?
(I understand when it comes to configs - cause anyone can have different configs and then there may be interference)
 
but when it comes to debug configuration - it is something that can be shared between all devs and it would only add some comfort cause you don't need to write it by yourself

I mean best practices exist for a reason but I just don't see it here in debug configurations being omitted  

(yep, for this case it contains envs. And also it is much easier to run tests when you have configured debug action, that's from personal experience)

Reply:
> (yep, for this case it contains envs. And also it is much easier to run tests when you have configured debug action, that's from personal experience)

How about calling `t.Setenv` in all test suites so that `go test ./...` command can succeed without requiring any env? It will also help the tests to be more specific to whether they require `DOCKER_HOST` or `SUPABASE_INTERNAL_API_HOST` only.

Reply:
@egor-romanov This is how I removed explicit env var from running tests https://github.com/supabase/cli/pull/283/files#diff-7f11b9ec624784dcb36925610adb78aa39e9c1e06bf867ddfba5bf99cb9b1f62R38. I believe `launch.json` is no longer necessary after this change.